cmd/snap/cmd_find.go

```diff
@@ -154,6 +176,18 @@ func (x *cmdFind) Execute(args []string) error {
 	}
 
 	cli := Client()
+
+	if x.Section != "" && x.Section != "featured" {
+		sections, err := getSections()
+		if err != nil {
+			return err
+		}
+		if !strutil.ListContains(sections, string(x.Section)) {
```

Comment:
Given that the sections file is sorted, I saw this and thought “could use `strutil.SortedListContains`”, but then I realised that the sections are sorted in `refreshCatalogs`, not in `getSections`. I'll move the sort and switch this to the more efficient search in a followup.